packages/components/Organisms/LinkTiles/LinkTiles.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <div class="rpl-link-tiles" v-bind:class="{ 'rpl-link-tiles--grid': gridView, 'rpl-link-tiles--wide': !sidebar}">
+    <h2 class="rpl-link-tiles__title">{{title}}</h2>
+    <ul class="rpl-link-tiles__list">
+      <li v-for="(link, index) in links" :key="index" class="rpl-link-tiles__list-item">
+        <rpl-link :href="link.url" :innerWrap="false" class="rpl-link-tiles__link">
+          <img v-if="!hideIcons && link.image" :src="link.image" alt="" class="rpl-link-tiles__link-icon"/>
+          <span>{{link.text}}</span>
+        </rpl-link>
+      </li>
+    </ul>
+    <rpl-link :href="primaryCta.url" class="rpl-link-tiles__primary-cta">{{primaryCta.text}}</rpl-link>
+  </div>
+</template>
+
+<script>
+// Add imports
+import RplLink from '@dpc-sdp/ripple-link'
+
+export default {
+  name: 'RplLinkTiles',
+  components: {
+    RplLink
+  },
+  props: {
+    title: String,
+    primaryCta: {
+      text: String,
+      url: String
+    },
+    links: Array,
+    hideIcons: { type: Boolean, default: false },
+    gridView: { type: Boolean, default: false },
+    sidebar: { type: Boolean, default: false }
+  }
+}
+</script>
+
+<style lang="scss">
+  @import "~@dpc-sdp/ripple-global/scss/settings";
+  @import "~@dpc-sdp/ripple-global/scss/tools";
+
+  $rpl-link-tiles-title-ruleset: (
+    'xs': ('l', 1.2em, 'bold'),
+    'm': ('xl', 1.2em, 'bold')
+  ) !default;
+  $rpl-link-tiles-tile-text-ruleset: (
+    'xs': ('xs', 1em, 'bold'),
+    'm': ('s', 1em, 'bold')
+  ) !default;
+  $rpl-link-tiles-item-background-color: rpl_color('white') !default;
+  $rpl-link-tiles-link-border: 1px solid rpl_color('mid_neutral_2') !default;
+  $rpl-link-tiles-link-shadow: 0 2px 8px 0 rgba(0, 0, 0, .08) !default;
+  $rpl-link-tiles-text-color: rpl_color('extra_dark_neutral') !default;
+  $rpl-link-tiles-text-hover-color: rpl_color('primary') !default;
+  $rpl-link-tiles-text-focus-outline-color: rpl_color('primary') !default;
+  $rpl-link-tiles-primary-cta-border-color: rpl_color('mid_neutral_2') !default;
+
+  .rpl-link-tiles {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-bottom: $rpl-space-4;
+
+    @include rpl_breakpoint(m) {
+      flex-direction: row;
+      align-items: center;
+      justify-content: space-between;
+      flex-wrap: wrap;
+    }
+
+    .rpl-link-tiles__list {
```

Comment:
@anfilimon Something we should update in our linting, but I''d prefer to use nested selectors here using `&__`  - see https://github.com/dpc-sdp/ripple/blob/develop/packages/components/Molecules/Alert/Alert.vue#L78 for example.

Reply:
Hey @dylankelly, I don't have permission to push to a Ripple branch but here's the commit in the fork - https://github.com/anfilimon/ripple/commit/88aab56988ee61b5d0c4eac9090972aecc845d60